src/data/faq_de.json

```diff
@@ -229,6 +229,7 @@
                         "anchor": "no_risk_update",
                         "active": true,
                         "textblock": [
+                            "Dieses Problem liegt normalerweise daran, dass einige Geräte-Anbieter eigene Batteriesparfunktionen implementieren, anstatt auf die bereits integrierten Funktionen zuzugreifen, die von Google in seinem Android-Betriebssystem bereitgestellt sind. Apps, wie die Corona-Warn-App, deren Sinn es ist, im Hintergrund Aufgaben zu erledigen, können auf solchen Geräten daran gehindert werden. D.h., dass eine nicht-funktionierende Hintergrundaktualisierung nicht auf die App selbst zurückzuführen ist, sondern auf spezifische Besonderheiten von einigen Geräten. Im Folgenden haben wir einige Maßnahmen zusammengestellt, das Problem zu beheben.",

```

Comment:
Android refers to the Battery of a mobile device as "Akku" in German.
CWA uses the Term "Akku" too .

So to be consistent with Android and the app, that should be "Akkusparkfunktionen" instead of "Batteriesparfunktionen".

Reply:
ok, thanks